pytransform3d/test/test_trajectories.py

```diff
@@ -72,6 +79,39 @@ def test_concat_many_to_one():
     assert_array_almost_equal(A2Cs, concat_many_to_one(A2Bs, B2C))
 
 
+def test_concat_dynamic():
+    rng = np.random.default_rng(84320)
+    A2Bs = np.stack([random_transform(rng) for _ in range(5)])
+    B2Cs = np.stack([random_transform(rng) for _ in range(5)])
+    A2Cs = concat_dynamic(A2Bs, B2Cs)
+    for i in range(len(A2Cs)):
+        assert_array_almost_equal(A2Cs[i], concat(A2Bs[i], B2Cs[i]))
+        assert_array_almost_equal(A2Cs[i], concat_dynamic(A2Bs[i], B2Cs[i]))
+    assert_array_almost_equal(concat_dynamic(A2Bs[0], B2Cs),
+                              concat_one_to_many(A2Bs[0], B2Cs))
+    assert_array_almost_equal(concat_dynamic(A2Bs, B2Cs[0]),
+                              concat_many_to_one(A2Bs, B2Cs[0]))
```

Comment:
Can you give a rough idea of your asserts in a comment? I have a vague idea that you want to check the "dynamic" version of concat against the "one-instance" version ...